packages/app/src/containers/LandingPage/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {
+    ImageBackground,
+    Image,
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import { Button } from "react-native-elements";
+
+interface LandingPageState {
```

Comment:
do we need this?

Reply:
@petermsouzajr  yes, we may need this in the future when we need to define local state for the container so i just created an empty interface for now.